PlutoHelperAgent/AppDelegate.m

```diff
@@ -132,11 +132,8 @@ - (void)tryOpenProject:(NSString *)projectid
                     if([finishedTask terminationStatus]!=0){
                         //[alert runModal] must be called on main thread. See https://stackoverflow.com/questions/4892182/run-method-on-main-thread-from-another-thread
                         dispatch_async(dispatch_get_global_queue(0, 0), ^{
-                            [alert setMessageText:@"Open script failed"];
-                            [alert setInformativeText:[NSString stringWithFormat:@"Couldn't open your project, because the open script returned error code %d.\n%@\n%@\n\nPlease contact multimediatech@theguardian.com and send a copy of this message",
-                                                       [finishedTask terminationStatus],
-                                                       [self getPipeData:stdOutPipe],
-                                                       [self getPipeData:stdErrPipe]
+                            [alert setMessageText:@"Opening Project Failed"];
```

Comment:
please could you add a line to log the actual error to the Console (`NSLog(@"")` I think), including the name of the script, arguments to the script and the error code.

Reply:
Done.